package.json

```diff
@@ -1,9 +1,8 @@
 {
   "devDependencies": {
-    "babel-core": "^6.22.1",
-    "babel-preset-env": "^1.2.2",
-    "babel-preset-latest": "^6.24.0",
-    "babel-register": "^6.22.0",
+    "@babel/core": "7.0.0-beta.42",
+    "@babel/preset-env": "7.0.0-beta.42",
+    "@babel/register": "7.0.0-beta.42",
```

Comment:
do you think it would be possible to use something like https://github.com/joliss/node-multidep to test this with both Babel versions?